tslint.json

```diff
@@ -1,9 +1,42 @@
 {
     "defaultSeverity": "error",
     "extends": [
-        "tslint:recommended"
+      "tslint:recommended",
+      "tslint-config-airbnb",
+      "tslint-react"
     ],
+    "linterOptions": {
+      "exclude": [
+        "node_modules/**",
+        "typings/**",
+        "App/Sagas/**/*",
+        "App/Redux/**/*",
+        "App/Navigation/**/*",
+        "App/Models/**/*",
+        "App/SB/**/*",
+        "**/*.json"
+      ]
+    },
     "jsRules": {},
-    "rules": {},
+    "rules": {
+      "no-empty": false,
+      "object-literal-sort-keys": false,
+      "object-shorthand-properties-first": false,
+      "space-before-function-paren": false,
+      "jsx-no-multiline-js": false,
+      "import-name": false,
+      "ordered-imports": false,
+      "member-access": false,
+      "variable-name": [true, "ban-keywords", "check-format", "allow-pascal-case", "allow-leading-underscore"],
+      "trailing-comma": [
+        true,
+        {
+          "multiline": "never",
+          "singleline": "never"
+        }
+      ],
+      "semicolon": [true, "never"],
+      "max-line-length": [true, 200]
```

Comment:
A lot of these rules probably aren't the greatest. But again, I wanted to get v0 of this PR in without having to change the entire project off the bat.